src/cluster_manager.sh

```diff
@@ -102,11 +112,27 @@ _add_node() {
         network_opts="${network_opts} --ip ${ip_address}"
     fi
 
+    local port_opts=""
+    local mount_opts=""
+    if [ "${EXPOSE_KUBEAPI_PORT}" = "1" ]; then
+        port_opts="-p 6443:6443"
+        local -r workdir=$(mktemp -d /tmp/kubeconfworkdirig-XXXXXX)
+        local -r config_file="${workdir}/microshift-config.yaml"
+        echo -e "apiServer:\n  subjectAltNames:\n    - $(_get_hostname)" > "${config_file}"
+        sudo podman volume create "${name}_config"
+        # create podman volume from the temp config file
+        tar -C "${workdir}" -cf  - "$(basename "${config_file}")" | sudo podman volume import "${name}_config" -
+        mount_opts="--volume ${name}_config:/etc/microshift/config.d/api:ro"
+        rm -rf "${workdir}"
```

Comment:
I'm still not sure this technique is justified because of a small text file.
What do  you think about creating config files in /var/lib/microshift-okd/config.d/ and sharing it with the pod using -v option? We only use this directory for `lvmdisk.image` files.